File: URI/1038.py
codigo, qtd = input().split()
codigo = int(codigo)
qtd = int(qtd)

if codigo == 1:
	print("Total: R$ {:.2f}".format(qtd*4.00))
elif codigo == 2:
	print("Total: R$ {:.2f}".format(qtd*4.50))
elif codigo == 3:
	print("Total: R$ {:.2f}".format(qtd*5.00))
elif codigo == 4:
	print("Total: R$ {:.2f}".format(qtd*2.00))
else:
	print("Total: R$ {:.2f}".format(qtd*1.50))

File: URI/1038.cpp
#include<iostream>
using namespace std;

int main(){
	float vet[] = {4.00,4.50,5.00,2.00,1.50};
	int N, qtd;

	cin >> N >> qtd;
	cout.precision(2);
	cout.setf(ios::fixed);
	cout << "Total: R$ " << vet[N-1] * qtd << endl;

	return 0;
}